.circleci/config.yml

```diff
@@ -150,19 +146,32 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - lint_and_test
+      - lint_and_test:
+          filters:
+            tags:
+              only: /v[\.0-9]+.*/
       - build:
           requires:
             - lint_and_test
+          filters:
+            tags:
+              only: /v[\.0-9]+.*/
       - bundle_test:
           requires:
             - build
+          filters:
+            tags:
+              only: /v[\.0-9]+.*/
       - vrt_test:
           requires:
             - build
       - publish:
-          filters:
-            tags:
-              only: /^v.*/
           requires:
             - bundle_test
```

Comment:
ah so bundle test needs filters because publish requires bundle test

Reply:
Yes, and that game bubbles all the way up.

Reply:
Given you do that you get a run with only those jobs when pushing a tag. The outcome it quite nice.